source/uwp/AdaptiveCardsObjectModel/idl/AdaptiveCards.ObjectModel.Uwp.idl

```diff
@@ -744,10 +768,7 @@ namespace AdaptiveCards.ObjectModel.Uwp
         String Label;
     };
 
-    [default_interface] runtimeclass AdaptiveTextBlockParser : IAdaptiveElementParser
-    {
-        AdaptiveTextBlockParser();
-    };
+    [default_interface] runtimeclass AdaptiveTextBlockParser : IAdaptiveElementParser { AdaptiveTextBlockParser(); };
```

Comment:
>[default_interface] runtimeclass AdaptiveTextBlockParser : IAdaptiveElementParser { AdaptiveTextBlockParser(); }; [](http://example.com/codeflow?start=4&length=113)

inconsistent with next interface

Reply:
This is what the auto-formatting is generating, so I'm going to leave as is. We can check with @paulcam206 when he gets back to see if there's a way for us to get this to turn out more consistent.